l10n_it_split_payment/models/account.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2015  Davide Corio <davide.corio@abstract.it>
+# Copyright 2015-2016  Lorenzo Battistini - Agile Business Group
+# Copyright 2016  Alessio Gerace - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountFiscalPosition(models.Model):
+    _inherit = "account.fiscal.position"
+
+    split_payment = fields.Boolean("Split Payment")
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    amount_sp = fields.Float(
+        string="Split Payment",
+        digits="Account",
+        store=True,
```

Comment:
Puoi dare un occhio a https://github.com/Borruso/l10n-italy/pull/5?

Reply:
non ho capito a cosa serve di preciso

Reply:
Installando un DB migrato, il campo viene ricreato perché non esiste nella tabella `account_move`.
Quando viene creato, viene anche eseguito il calcolo del suo valore ma non serve perché tutti i valori esistono già nella tabella `account_invoice`.
Inoltre, può sollevare errori tipo
> Impossibile aggiungere/modificare registrazioni antecedenti o pari alla data di chiusura 24/09/2020.

Lo script di migrazione che aggiungo in https://github.com/Borruso/l10n-italy/pull/5 serve a copiare i valori da `account_invoice` a `account_move` in modo da non eseguire il ricalcolo.